courses/machine_learning/deepdive2/supplemental/solutions/object_detection_with_automl.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<ul><li>put the import at the beginning of the notebook</li><li>Instead of redefining a <code>project_id</code> variable here, reuse the one you defined above (i.e.,<code>PROJECT_ID`)﻿</code></li><li>"display_name" -&gt; "DATASET_NAME" (usually constants are defined in upper case)</li><li>Set the actual value of <code>DATASET_NAME</code> to something more informative that "dataset_name"</li><li>Instead of hardcoding "us-central1", reuse the <code>ZONE</code> variable you defined earlier</li></ul><p> </p><ul><li>Add some explanation before this cell explaining what this snippet of code is doing</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>done</p>